### PR TITLE
tls-decrypt-exit-on-error

### PR DESCRIPTION
**Pull request by calvix**
## Checklist

- [x] Update changelog in CHANGELOG.md.

towards https://github.com/giantswarm/giantswarm/issues/12808

### CODE REVIEW

File: service/internal/cloudconfig/template/decrypt_keys_assets_script.go [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package template
 
 const DecryptKeysAssetsScript = `#!/bin/bash -e
+set -o errexit
```
Comment (njuettner) [inline service/internal/cloudconfig/template/decrypt_keys_assets_script.go:4]: would this be the same as if you would set `set -e`

Comment (calvix) [reply service/internal/cloudconfig/template/decrypt_keys_assets_script.go:4]: True, but this is at least little bit more decriptive

Comment (njuettner) [reply service/internal/cloudconfig/template/decrypt_keys_assets_script.go:4]: Thanks, I haven't seen the longer version of it, that's why I was asking 👍

**Comment by calvix** [comment]
tested on gauss, looks good

**Pull request merged**